ironfish/src/workerPool/serializableMessage.ts

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import bufio from 'bufio'
+import { Serializable } from '../common/serializable'
+
+export enum WorkerMessageType {
```

Comment:
We can leave this empty for now and add values as each job is refactored over.

Reply:
`serializeWithMetadata()` depends on the string values for enum members in order to write a VarString to a Buffer. If we make this empty for now, perhaps we should remove the `bw.writeVarString(this.type)` call for now, and then add it back once we start implementing each task.

Reply:
Yeah let's omit it for now so it'll be more explicit for subsequent PRs.

Reply:
Changed.